tools/mass_perldoc_md_creation.pl

```diff
@@ -0,0 +1,176 @@
+derived from the documentation of the perl scripts and libraries are updated.
+
+If any new script have been added to a given release, make sure to include it
+in the variable called @script_list at the beginning of the script.
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+LORIS community <loris.info@mcin.ca> and McGill Centre for Integrative
+Neuroscience
+
+=cut
+
+
+use strict;
+use warnings;
+
+use File::Basename;
+use Getopt::Tabular;
+
+
+use NeuroDB::DBI;
+use NeuroDB::ExitCodes;
+
+my @script_list = (
+    'DTIPrep/DTI/DTI.pm',
+    'DTIPrep/DTIPrepRegister.pl',
+    'DTIPrep/DTIPrep_pipeline.pl',
+    'batch_uploads_imageuploader',
+    'batch_uploads_tarchive',
+    'dicom-archive/DICOM/DCMSUM.pm',
+    'dicom-archive/DICOM/DICOM.pm',
+    'dicom-archive/DICOM/Element.pm',
+    'dicom-archive/DICOM/Fields.pm',
+    'dicom-archive/DICOM/Private.pm',
+    'dicom-archive/DICOM/VRfields.pm',
+    'dicom-archive/dicomTar.pl',
+    'dicom-archive/updateMRI_Upload.pl',
+    'tools/BackPopulateSNRAndAcquisitionOrder.pl',
+    'tools/MakeArchiveLocationRelative.pl',
+    'tools/ProdToConfig.pl',
+    'tools/database_files_update.pl',
+    'tools/mass_perldoc_md_creation.pl',
+    'tools/seriesuid2fileid',
+    'uploadNeuroDB/NeuroDB/DBI.pm',
+    'uploadNeuroDB/NeuroDB/ExitCodes.pm',
+    'uploadNeuroDB/NeuroDB/File.pm',
+    'uploadNeuroDB/NeuroDB/FileDecompress.pm',
+    'uploadNeuroDB/NeuroDB/ImagingUpload.pm',
+    'uploadNeuroDB/NeuroDB/MRI.pm',
+    'uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm',
+    'uploadNeuroDB/NeuroDB/Notify.pm',
+    'uploadNeuroDB/bin/concat_mri.pl',
+    'uploadNeuroDB/bin/mincpik',
+    'uploadNeuroDB/imaging_upload_file.pl',
+    'uploadNeuroDB/imaging_upload_file_cronjob.pl',
+    'uploadNeuroDB/mass_nii.pl',
+    'uploadNeuroDB/mass_pic.pl',
+    'uploadNeuroDB/minc_deletion.pl',
+    'uploadNeuroDB/minc_insertion.pl',
+    'uploadNeuroDB/register_processed_data.pl',
+    'uploadNeuroDB/tarchiveLoader',
+    'uploadNeuroDB/tarchive_validation.pl'
+);
+
+
+my $profile;
+my $verbose = 0;
+
+my $profile_desc = "name of config file in ../dicom-archive/.loris_mri";
+
+my @opt_table = (
+    [ "-profile", "string",  1, \$profile, $profile_desc ],
+    [ "-verbose", "boolean", 1, \$verbose, "Be verbose." ]
+);
+
+my $Help = <<HELP;
+******************************************************************************
+Creates the perldoc .md files stored in LORIS-MRI/docs/scripts_md/
+******************************************************************************
+
+This script will need to be run once per release to make sure the .md files
+derived from the documentation of the perl scripts and libraries are updated.
+If any new script have been added to a given release, make sure to include it
+in the variable called \@script_list at the beginning of the script.
+
+Documentation: perldoc mass_perldoc_md_creation.pl
+
+HELP
+my $Usage = <<USAGE;
+usage: tools/mass_perldoc_md_creation.pl -profile \$profile
```

Comment:
perhaps add the word `perl` before `tools/mass...`

Reply:
done